.github/workflows/sync_models.yml

```diff
@@ -0,0 +1,32 @@
+# Reference from:
+# https://github.com/c-bata/go-prompt/blob/master/.github/workflows/test.yml
+name: sync models from k8s openapi
+on: workflow_dispatch
+jobs:
+  sync-k8s-api:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - name: Set up Go 1.18
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.18
+      - name: build-kcl-openapi
+        run: go build
+      - name: sync k8s models
+        run: |
+          tags=("1.28" "1.29" "1.30" "1.31" "1.32" "1.33")
+          for tag in "${tags[@]}"
+          do
+            spec_path=swagger.json
+            rm -f ${spec_path}
+            wget https://raw.githubusercontent.com/kubernetes/kubernetes/release-${tag}/api/openapi-spec/swagger.json -O ${spec_path}
+            python ./scripts/preprocess/main.py ${spec_path} --omit-status --rename=io.k8s=k8s
+            ./kcl-openapi generate model -f processed-${spec_path}
```

Comment:
I noticed that the title of the relevant Jira transaction states that the model still needs to be uploaded. How do we upload the model here, or is this PR only used to complete the testing function of model generation.